tests/functional/firefox/os/version/test_base.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.os.version.base import FirefoxOSBasePage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('version', [('2.5'), ('2.0'), ('1.4'), ('1.3t'), ('1.3'), ('1.1')])
```

Comment:
Do we want all versions to run in the sanity suites? If not, you can mark individual parameters:

```python
@pytest.mark.parametrize('version', [
    pytest.mark.sanity(('2.5')),
    pytest.mark.sanity(('2.0')),
    ('1.4'),
    ('1.3t'),
    ('1.3'),
    ('1.1')])
```